bot/exts/moderation/infraction/_utils.py

```diff
@@ -75,7 +71,7 @@ async def post_user(ctx: Context, user: UserSnowflake) -> t.Optional[dict]:
 
 async def post_infraction(
         ctx: Context,
-        user: UserSnowflake,
+        user: MemberOrUser,
```

Comment:
These `getattr` can be replaced with normal attribute access

https://github.com/python-discord/bot/blob/8ae21a26e4c74314d0726a37e71877f8566fe6a7/bot/exts/moderation/infraction/_utils.py#L56-L59

Reply:
Fixed locally :+1: